CHANGELOG.md

```diff
@@ -84,13 +92,43 @@ This release made possible by the awesome work of the following contributors (in
 
 <3
 
-### Support ###
+## 0.3
+
+### [0.3.1] - 2017-05-19
+
+KubeSpawner updates.
+
+### [0.3] - 2017-05-15
+
+Deployer UX fixes.
+
+## [0.2] - 2017-05-01
+
+Minor cleanups and features.
+
+## [0.1] - 2017-04-10
+
+Initial Public Release
+
+## Support
 
 If you need support, reach out to us on
 [gitter](https://gitter.im/jupyterhub/jupyterhub) or open an
 [issue](https://github.com/jupyterhub/helm-chart/issues).
 
-### About this cricketer ###
+## About the Cricketers
+
+### Akram
```

Comment:
I'd like to move this under the release itself, so it isn't lost in a separate section you don't get to until you are at the end